examples/nirfmxwcdma/acp-single-carrier.py

```diff
@@ -0,0 +1,333 @@
+r"""Fetch ACP data.
+
+Steps:
+1. Open a new RFmx Session.
+2. Configure Frequency Reference.
+3. Configure basic signal properties (Center Frequency, RF Attenuation and External Attenuation)
+4. Configure Trigger Type and Trigger Parameters.
+5. Configure Reference Level.
+6. Select ACP measurement and enable Traces.
+7. Configure Measurement Method.
+8. Configure Averaging Parameters for ACP measurement.
+9. Configure Sweep Time Parameters.
+10. Configure Noise Compensation Parameter.
+11. Configure Number of offset channels.
+12. Initiate the Measurement.
+13. Fetch ACP Measurements and Traces.
+14. Close RFmx Session.
+
+
+The gRPC API is built from the C API. RFmx WCDMA documentation is installed with the driver at:
+C:\Program Files (x86)\National Instruments\RFmx\WCDMA\Documentation\wcdmacvi.chm
+
+Getting Started:
+
+To run this example, install "RFmx WCDMA" on the server machine.
+Link: https://www.ni.com/en-us/support/downloads/software-products/download.rfmx-wcdma.html
+
+For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a
+gRPC Client" wiki page.
+Link: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+
+Refer to the NI-RFmxWCDMA gRPC Wiki for the latest C Function Reference:
+Link: https://github.com/ni/grpc-device/wiki/NI-RFmxWCDMA-C-Function-Reference
+
+Running from command line:
+
+Server machine's IP address, port number, and physical channel name can be passed as separate
+command line arguments.
+  > python acp-single-carrier.py <server_address> <port_number> <physical_channel_name>
+If they are not passed in as command line arguments, then by default the server address will be
+"localhost:31763", with "SimulatedDevice" as the resource name
+"""
+
+import sys
+
+import grpc
+import nirfmxwcdma_pb2 as nirfmxwcdma_types
+import nirfmxwcdma_pb2_grpc as grpc_nirfmxwcdma
+
+server_address = "localhost"
+server_port = "31763"
+session_name = "RFmxWCDMASession"
+
+# Resource name and options for a simulated 5663 client.
+resource = "SimulatedDevice"
+options = "Simulate=1,DriverSetup=Model:5663"
+
+# Read in cmd args
+if len(sys.argv) >= 2:
+    server_address = sys.argv[1]
+if len(sys.argv) >= 3:
+    server_port = sys.argv[2]
+if len(sys.argv) >= 4:
+    resource = sys.argv[3]
+    options = ""
+
+# Create a gRPC channel + client.
+channel = grpc.insecure_channel(f"{server_address}:{server_port}")
+client = grpc_nirfmxwcdma.NiRFmxWCDMAStub(channel)
+instr = None
+
+
+# Maximum size of an error message
+NUMBER_OF_OFFSETS = 2
+
+
+def raise_if_error(response):
```

Comment:
Can you update this to follow the other examples like [here for LTE](https://github.com/ni/grpc-device/blob/2b3bcc8774ae89de5afa378380bea84e2c01969f/examples/nirfmxlte/acp-single-carrier.py#L76)? 

The main try catch should handle API errors but we can have a `check_for_warning` helper like the other examples to handle if a warning is returned from a call.

Reply:
Thanks. In the new commit, I fixed this issue by using check_for_warning helper.